### ISSUE TITLE
Actor conditions - display rounds left

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#212]
```
tek stated here:
http://andors.techby2guys.com/viewtopic.php?f=4&t=862

It would be really helpful if the number of rounds left on the active actor 
conditions appeared above the actor condition icon on the playing screen. When 
I get the fatigue condition I like to see when it will end. It is getting 
really annoying having to keep going to the "overview" screen to see the # of 
rounds left.

I stated here:
http://andors.techby2guys.com/viewtopic.php?p=2763#p2763

There is no description about the actor conditions anywhere. IMO all actor 
conditions that are given by items or used by monsters should be transparent. 
Currently I cannot see something like this.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 7:06

**Comment by GoogleCodeExporter** [comment]
```
Please try to separate different reports into separate issues.

The second report is now separated into issue 217.

Updating this issue to only reflect the suggestion from tek about displaying 
number of rounds left on active actor conditions.

Thanks for the report.
```

Original comment by `oskar.wi...@gmail.com` on 15 Jul 2011 at 4:46
- Changed title: **Actor conditions - display rounds left**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Magnitudes are now shown on actor condition icons, released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:39
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.12**
